dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -300,6 +300,15 @@ public class ConnectorArguments extends DefaultArguments {
           .withValuesSeparatedBy(',')
           .describedAs("logTable,queryTable");
 
+  private final OptionSpec<String> optionUtilityLogAlternate =
+      parser
+          .accepts(
+              "utility-log-alternate",
+              "the alternate utility log table to export (teradata-logs only)")
```

Comment:
Since this is Teradata only, should this be called `--teradata-utility-log-alternate`?

Reply:
I have renamed the option to `-Dteradata-logs.utility-logs-table`.